src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -32,9 +29,12 @@ private static void setupPresetTests(Application application) {
                 DriverFeature.getDriverManager());
         SelectTestFigure2OptionListener selectTestFigure2OptionListener = new SelectTestFigure2OptionListener(
                 DriverFeature.getDriverManager());
+        SelectTestMouseListener selectTestMouseListener = new SelectTestMouseListener(
+                DriverFeature.getDriverManager(), application);
 
         application.addTest("Figure Joe 1", selectTestFigureOptionListener);
         application.addTest("Figure Joe 2", selectTestFigure2OptionListener);
+        application.addTest("Mouse Listener", selectTestMouseListener);
```

Comment:
"Listener" is an implementational detail. Please find a more adequate description (e.g., "Enable mouse control")